File: laramie-ingresos-publicos-webapi/src/domain/entities/provincia.ts
export default class Provincia {

    id: number;
	codigo: string;
	nombre: string;
	orden: number;
	idPais: number;

	constructor(
        id: number = 0,
		codigo: string = "",
		nombre: string = "",
		orden: number = 0,
		idPais: number = 0
	)
	{
        this.id = id;
		this.codigo = codigo;
		this.nombre = nombre;
		this.orden = orden;
		this.idPais = idPais;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.codigo = row.codigo ?? "";
		this.nombre = row.nombre ?? "";
		this.orden = row.orden ?? 0;
		this.idPais = row.idPais ?? 0;
	}

}

File: modules/administracion/models/provincia.py
from sqlalchemy import Column, BigInteger, String, Integer, ForeignKey

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Provincia(Base):
    __tablename__ = "administracion_provincias"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    codigo = Column(String(50), nullable=False)
    nombre = Column(String(250), nullable=False)
    orden = Column(Integer, nullable=False, default=0)
    id_pais = Column(BigInteger, ForeignKey("administracion_paises.id"), nullable=False)
